src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double height;
+    private double firstLeg;
+    private double secondLeg;
+
+    public IsoscelesTrapezoid(String color, double height, double firstLeg, double secondLeg) {
+        super(color);
+        this.height = height;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double calculateArea() {
+        return ((firstLeg + secondLeg) / 2) * height;
+    }
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The draw() and calculateArea() methods are not conceptually close to each other, so calculateArea() should be in a separate interface.